static/js/transmit.js

```diff
@@ -15,7 +22,8 @@ exports.send_message = function (request, on_success, error) {
             sent_messages.report_server_ack(request.local_id);
         },
         error(xhr, error_type) {
-            if (error_type !== "timeout" && reload_state.is_pending()) {
+            if (error_type !== "timeout" || reload_state.is_pending()) {
+                compose_state.message_content(saved_message_content);
```

Comment:
fab64fd7b0e78d7464f91003fb7d49007a9478b7 appears to where this bit of `&&` logic is introduced.  I don't think I understand the reasoning behind why we want an || gate here, though -- maybe worth adding a comment above this?

I **think** the idea is that if we're in the process of doing a reload, we guess -- but what does `error_type !== "timeout"` mean?